<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWdiget</class>
 <widget class="QWidget" name="MainWdiget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="2,0">
     <item>
      <widget class="QLCDNumber" name="lcdNumber"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="startBtn">
         <property name="text">
          <string>开始计时</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pauseBtn">
         <property name="text">
          <string>暂停计时</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopBtn">
         <property name="text">
          <string>停止计时</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
